packages/fluentui/react-northstar/src/themes/teams/components/Loader/loaderVariables.ts

```diff
@@ -32,6 +34,7 @@ export const loaderVariables = (): LoaderVariables => ({
   },
 
   svgContent: loaderSvgDataUrl,
+  svgSecondaryContent: loaderSecondarySvgDataUrl,
```

Comment:
```suggestion
  secondarySvgContent: loaderSecondarySvgDataUrl,
```

Accordion to naming convention it prop should be first